src/Bladeburner/Bladeburner.ts

```diff
@@ -750,32 +751,20 @@ export class Bladeburner {
     }
   }
 
+  public killRandomSupportingSleeves(n: number) {
+    const sup = [...Player.sleevesSupportingBladeburner()]; // Explicit shallow copy
+    shuffleArray(sup);
+    sup.slice(0, Math.min(sup.length, n)).forEach((sleeve) => sleeve.kill());
```

Comment:
Optional: I find it easier to read the mutation action in non-functional style with `for..of`, where the `sleeve.kill()` would end up on its own line instead of all mushed together with `forEach`, but it's more of a preference thing.

Reply:
I'll skip this, possibly reviewing on a second pass (different PR).